app/usecases/sendAbsent.js

```diff
@@ -2,16 +2,16 @@ const gplaces = require("../services/gplaces")();
 const mailer = require("../services/mailer")();
 const watsonSpeech = require("../services/watsonSpeech")();
 const lectureCalId="1nc6dpksqqc9pk2jg85f0hd5hkn8ups1@import.calendar.google.com"; // TO-DO Replace with Preference of lecture calendar
```

Comment:
Funktioniert das auf Bengins Server immer noch?
@xdivby0

Reply:
Sollte das in die .env?

Reply:
das wird wie gesagt sobald das geht von den preferences geholt

Reply:
was genau meinst du?